Tests/js/views/fields/assets/ez-richtext-editview-tests.js

```diff
@@ -612,6 +559,28 @@ YUI.add('ez-richtext-editview-tests', function (Y) {
             this.wait();
         },
 
+        "Should disable the focus mode on tap on the save and return button": function () {
+            var that = this,
+                button = this.view.get('container').one('.ez-richtext-save-and-return');
+
+            this.view.set('focusMode', true);
+            this.view.get('container').once('tap', function (e) {
+                that.resume(function () {
+                    Assert.isTrue(
+                        !!e.prevented,
+                        "The tap event should be prevented"
+                    );
+                    Assert.isFalse(
+                        this.view.get('focusMode'),
+                        "The focus mode should be disabled"
+                    );
+                });
+            });
+
+            button.simulateGesture('tap');
+            this.wait();
+        },
+
         "Should disbale the focus mode on saveReturnAction event": function () {
```

Comment:
because you also need to remove https://github.com/StephaneDiot/PlatformUIBundle-1/blob/EZP-24829_Change_the_display_of_the_richtext_focus_mode/Resources/public/js/views/fields/ez-richtext-editview.js#L55